web/http/client.zig

```diff
@@ -1,12 +1,17 @@
 const std = @import("std");
 
 test "Status == 200" {
-    const uri = std.Uri.parse("https://ziglang.org/") catch unreachable;
+    const uri = std.Uri.parse("https://ziglang.org") catch unreachable;
     var client: std.http.Client = .{ .allocator = std.testing.allocator };
     defer client.deinit();
+    var h = std.http.Headers{ .allocator = std.testing.allocator };
+    defer h.deinit();
 
-    var req = try client.request(uri, .{}, .{});
+    var req = try client.request(.GET, uri, h, .{});
     defer req.deinit();
-
-    try std.testing.expect(req.response.headers.status == .ok);
+    try req.start(); // start request
+    try req.finish(); // end request
```

Comment:
This is a noop, there's no payload to finish.

Reply:
Nice. Thanks for the suggestions, I'll make the planned fixes tomorrow.
However, if you want to send a PR, you are welcome.